gulpfile.js/tasks/pug.js

```diff
@@ -48,7 +48,7 @@ var pugTask = function () {
         .on('error', handleErrors)
         .pipe(gulpif(global.production, htmlmin(pugConfig.htmlmin)))
         .pipe(gulp.dest(paths.dest))
-        .pipe(browserSync.stream())
+        .on('end', browserSync.reload)
```

Comment:
Haven't tried this yet, but could this have been fixed instead with `.pipe(browserSync.stream({once: true}))` instead?